simulation.cpp

```diff
@@ -234,7 +235,7 @@ double calculate_optimal(const simulation &s)
 std::vector<double> create_inputs(simulation s)
 {
     environment &e = s.get_env();
-    return(create_n_inputs(e, get_inds_input_size(s), s.get_rng() ));
```

Comment:
why not keep this free function?

Reply:
I think I changed that with the previous implementation because the size of the inputs in simulation and individuals were not always equal, I can revert it.